OracleManagementAgent/README.md

```diff
@@ -1,107 +1,129 @@
 
-1. Copy the downloaded bundle to the same directory as the Dockerfile
+1. Copy the downloaded bundle to the same directory as the `Dockerfile`
 
     ```shell
+    # stage install bundle
     $ cp oracle.mgmt_agent.zip OracleManagementAgent/dockerfiles/latest/
     ```
 
-1. Follow the steps in the [Create Install Key](https://docs.oracle.com/en-us/iaas/management-agents/doc/management-agents-administration-tasks.html#GUID-C841426A-2C32-4630-97B6-DF11F05D5712) and [Configure a Response File](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html#GUID-5D20D4A7-616C-49EC-A994-DA383D172486) sections of the [Management Agent](https://docs.oracle.com/en-us/iaas/management-agents/index.html) documentation to create an install key and save it locally as `input.rsp`.
-
-1. Copy the downloaded install key to the same directory as the Dockerfile
+1. Change to the directory in which the `Dockerfile` for this container image is located
 
     ```shell
-    $ cp input.rsp OracleManagementAgent/dockerfiles/latest/
+    # to build and run container from this location
+    $ cd OracleManagementAgent/dockerfiles/latest/
     ```
-
-1. Change to the directory in which the `Dockerfile` for this container image is located
+<!-- markdownlint-disable MD013 -->
+1. Follow the steps in the [Create Install Key](https://docs.oracle.com/en-us/iaas/management-agents/doc/management-agents-administration-tasks.html#GUID-C841426A-2C32-4630-97B6-DF11F05D5712) and [Configure a Response File](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html#GUID-5D20D4A7-616C-49EC-A994-DA383D172486) sections of the [Management Agent](https://docs.oracle.com/en-us/iaas/management-agents/index.html) documentation to create an install key and save it locally as `input.rsp` in the current directory.
+<!-- markdownlint-enable MD013 -->
+1. Create a bind mount location to persist container data
 
     ```shell
-    $ cd OracleManagementAgent/dockerfiles/latest/
+    # Important: cleanup any old state files
+    $ rm -rf /home/$USER/oracle
+    $ mkdir -p /home/$USER/oracle
     ```
 
 1. Ensure your tenancy is configured correctly by [applying the documented prerequisites for deploying management agents](https://docs.oracle.com/en-us/iaas/management-agents/doc/perform-prerequisites-deploying-management-agents.html)
 
-#### Steps to build and run using Docker Compose
+## Steps to build and run using Docker Compose
+
+1. Create .env file to populate the environment variables
 
-1. Create .env file to populate the hostname variable
     ```shell
+    # required environment settings
     $ echo "mgmtagent_hostname=mgmtagent912" > .env
+    $ echo "DOCKER_BASE_DIR=$(/home/$USER/oracle)" >> .env
+    $ echo "USERID=$(id -u)" >> .env
+    $ echo "GROUPID=$(id -g)" >> .env
     ```
-    **Note: Chose a unique hostname as it will be used to identify Management Agent in the UI.**
+
+    **Notes:**
```

Comment:
tried this but it indents the Notes section to appear as if it doesn't belong to the previous item

Reply:
They don't belong to the previous item. The notes here and below are identical and thus should be elevated to their own section instead of being repeated.